lib/browser.ts

```diff
@@ -184,7 +185,7 @@ export class Protractor {
    *
    * @type {Plugins} Object containing plugin funtions from config.
    */
-  private plugins_: Plugins;
+  plugins_: Plugins;
```

Comment:
They were marked as private; however, [lib/browser.ts](https://github.com/angular/protractor/blob/9b48143f90077b2d02f70d0dff19c376bb4c6824/lib/runner.ts#L199) makes a call to a private variable. This is the same issue for all the private variables in browser. They probably should be using a Get method.